cmd/clusterctl/main.go

```diff
@@ -17,27 +17,13 @@ limitations under the License.
 package main
 
 import (
-	"flag"
-
 	"k8s.io/klog"
 	"sigs.k8s.io/cluster-api-provider-gcp/pkg/cloud/google"
 	"sigs.k8s.io/cluster-api/cmd/clusterctl/cmd"
 	"sigs.k8s.io/cluster-api/pkg/apis/cluster/common"
 )
 
-// initLogs is a temporary hack to enable proper logging until upstream dependencies
-// are migrated to fully utilize klog instead of glog.
-func initLogs() {
-	flag.Set("logtostderr", "true")
-	flags := flag.NewFlagSet("klog", flag.ExitOnError)
-	klog.InitFlags(flags)
-	flags.Set("alsologtostderr", "true")
-	flags.Set("v", "4")
-	flag.Parse()
-}
-
 func main() {
-	initLogs()
 	var err error
```

Comment:
We don't need klog.InitFlags(nil) here?

Reply:
The flags are already initialized upstream as part of `cmd.Execute`.